packages/renderer/src/lib/table/TestTable.svelte

```diff
@@ -22,19 +20,22 @@ const people: Person[] = [
 
 const idCol: Column<Person> = new Column('Id', {
   align: 'right',
-  renderer: TestColumnId,
+  renderMapping: obj => obj.id,
+  renderer: SimpleColumn,
   comparator: (a, b) => a.id - b.id,
 });
 
 const nameCol: Column<Person> = new Column('Name', {
   width: '3fr',
-  renderer: TestColumnName,
+  renderMapping: obj => obj.name,
+  renderer: SimpleColumn,
   comparator: (a, b) => a.name.localeCompare(b.name),
 });
 
 const ageCol: Column<Person> = new Column('Age', {
   align: 'right',
-  renderer: TestColumnAge,
+  renderMapping: obj => obj.age,
+  renderer: SimpleColumn,
```

Comment:
Likewise I agree this would be good (especially if there was a way to tie in what type the renderer supports, should be either Type or this NewType ...) but I can't find similar examples, and my attempts at what I think might be valid syntax aren't working.

Reply:
Did you mean like changing it to `renderer: { component: SimpleColumn, mapping: obj => obj.name }` where second is optional? I was hoping there was something simpler when you don't need the mapping but haven't found any other options.

Reply:
I'm fine for now as it's not an exposed API we can refactor whenever we want and see if we would really benefit from having a single entry rather than two

Reply:
I don't think it makes a huge difference, but since they are related it probably makes sense. I'd prefer not to touch it while there are two PRs in review it would 'break', so I've opened #5071 for me to follow up on this later.